utils/compiler/src/PhpScoper/StaticEasyPrefixer.php

```diff
@@ -16,6 +16,8 @@ final class StaticEasyPrefixer
         'Doctrine\Inflector\Inflector',
         // for SmartFileInfo
         'Symplify\SmartFileSystem\SmartFileInfo',
+        // for ComposerJson because it is part of the public API. I.e. ComposerRectorInterface
+        'Symplify\ComposerJsonManipulator\ValueObject\ComposerJson'
```

Comment:
This should be the only change.
Scoper is aliasing classes, so the 2 cases above should be delegated corretly.

Reply:
Yes, was not sure about it. Because you did something for the SmartFileInfo.